microsoft/fluentui-system-icons
# Add Flutter support to the FluentIcons library

This PR is to add flutter support to the FluentIcons package
Issue: #34 

Steps to test:

Windows/Mac
1. Install Android Studio 
2. Install flutter for Windows/Mac : https://flutter.dev/docs/get-started/install
3. In Android Studio add Flutter plugin 
        a. Configure -> Plugin -> Search for flutter
4. Create a sample flutter application to make sure that everything is setup correctly.
5. Clone the project , switch to the flutter branch
6. open fluentui-system-icons as an existing Android project
7. For testing on Android  : ->Create an Android emulator or actual android device 
8. For testing on iOS: -> Create an iOS simulator , in your device section in Android studio , select the iOS simulator and run the app.
